changelog/db.changelog-1.3.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="DEFRA" id="1596009647200-1">
+      <tagDatabase tag="v1.1.0" />
+    </changeSet>
+    <changeSet author="liquibase (generated)" id="4">
```

Comment:
We cant change this now as its already been deployed to SND but just a note to change the `author` field to 'Defra' or 'Defra devs' etc for future liquibase change sets

Reply:
Ah yeah I forgot the author was one of the fields we were supposed to change. Will note this down, thank you.